yarn.lock

```diff
@@ -11310,30 +11310,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"protobufjs@npm:^6.10.0":
```

Comment:
What are the steps you took to do this @nawatts?

Reply:
I manually deleted this `"protobufjs@npm:^6.10.0":` entry from `yarn.lock` and ran `yarn` to update `.pnp.cjs` and the cache.

Reply:
Good to know. Sorry I missed this.